pkg/karmadactl/cmdinit/utils/rbac.go

```diff
@@ -0,0 +1,89 @@
+package utils
+
+import (
+	"context"
+	"fmt"
+
+	rbacv1 "k8s.io/api/rbac/v1"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/klog/v2"
+)
+
+// ClusterRoleFromRules ClusterRole Rules
+func ClusterRoleFromRules(name string, rules []rbacv1.PolicyRule, labels map[string]string) *rbacv1.ClusterRole {
+	return &rbacv1.ClusterRole{
+		TypeMeta: metav1.TypeMeta{
+			APIVersion: "rbac.authorization.k8s.io/v1",
+			Kind:       "ClusterRole",
+		},
+		ObjectMeta: metav1.ObjectMeta{
+			Name:   name,
+			Labels: labels,
+		},
+		Rules: rules,
+	}
+}
```

Comment:
Just a question: Do you think this kind of function `common`  enough?

Reply:
Both packages `pkg/karmadactl/cmdinit/kubernetes`  and `pkg/karmadactl/cmdinit/karmada` use this function, and its implementation has basically satisfied the general purpose , so I think this is enough `common`